src/controllers/house.controllers.js

```diff
@@ -11,5 +11,12 @@ const createNewHouseController = async (houseData) => {
     return { status: 'invalid', message: err };
   }
 };
-
+export const getById = async (id) => {
+  try {
+    const favorite = await favoriteModel.findById(id);
+    return { status: 'success', favorite };
+  } catch (err) {
+    return { status: 'invalid', message: 'There is no favorite with this ID' };
+  }
+};
```

Comment:
The favorite model should be only used by the favorite controller and then if you want to use it you should import the necessary function from the favorite controller.

Reply:
You can use `findByIdAndUpdate()` function.
I think that this task needs the house's status changed and not only removed from favorites.